src/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.tsx

```diff
@@ -1,7 +1,105 @@
+import { useTranslation } from 'react-i18next'
 import { Box } from '@mui/material'
+import Typography from '@mui/material/Typography'
+import Switch from '@mui/material/Switch'
+
+import useForm from '~/hooks/use-form'
+import { styles } from '~/containers/my-quizzes/quiz-settings-container/QuizSettingsContainer.styles'
+import { defaultStyles } from '~/components/app-content-switcher/AppContentSwitcher.styles'
 
 const QuizSettingsContainer = () => {
-  return <Box>Quiz settings</Box>
+  const { t } = useTranslation()
+  const { data, handleInputChange } = useForm({
+    initialValues: {
+      pointValues: false,
+      scoredUnscoredResponses: false,
+      correctAnswers: false,
+      shuffleQuestions: false
+    }
+  })
+
+  return (
+    <Box>
+      <Box>
+        <Typography sx={{ ...styles.title, ...styles.topTitle }}>
+          {t('myResourcesPage.quizzes.settingsPointsAndAnswers')}
+        </Typography>
+
+        <Box sx={styles.settingContainer}>
+          <Box>
+            <Typography sx={styles.subtitle}>
+              {t('myResourcesPage.quizzes.pointValues')}
+            </Typography>
+            <Typography sx={styles.description}>
+              {t('myResourcesPage.quizzes.pointValuesDesc')}
+            </Typography>
+          </Box>
+          <Switch
+            checked={data.pointValues}
+            data-testid='pointValues-switch'
+            onChange={handleInputChange('pointValues')}
+            sx={defaultStyles.switch}
+          />
+        </Box>
```

Comment:
How do you think, maybe we can make reusable component out of this to not repeat the same pieces of code?

Reply:
> I think .map it's an overkill decision for rendering two different titles and four switches at all.

Yeah, you are right, but I'm talking just about using this newly crafted item, without a `.map` function. Something like this:

```js
<Box>
    <SettingItem propsHere/>
    <SettingItem propsHere/>
    <SettingItem propsHere/>
</Box>
```

What do you think?

Reply:
U right, thank you, I made changes